repositories.yaml

```diff
@@ -123,3 +123,28 @@ firefox-reality:
   dependencies:
     - org.mozilla.components:service-glean
     - org.mozilla.components:support-sync-telemetry
+lockwise-ios:
+  app_id: org.mozilla.ios.Lockbox
+  notification_emails:
+    - frank@mozilla.com
+    - tlong@mozilla.com
+    - lockwise-dev@mozilla.com
+  url: 'https://github.com/mozilla-lockwise/lockwise-ios'
+  dependencies:
+    - org.mozilla.components:service-glean
+firefox-desktop:
```

Comment:
The fogotype is a "burn after using" prototype. Will attaching it to the "firefox-desktop" name here cause naming collisions down the line when we remove this one after fogotype's end, and want to add it back when we integrate the Glean SDK for real?

Reply:
It will indeed cause naming collisions. If this is a prototype let's use a different name. @chutten opinions on what it should be? `fog` seems short and unclear.

Reply:
`fogotype` works for me, as that's the "application" (heavy use of airquotes) that's sending the "prototype" pings with the app id `org.mozilla.fogotype`. Sound good, @fbertsch ?